iModelCore/BeSQLite/BeSQLite.cpp

```diff
@@ -5427,11 +5429,14 @@ static void initLanguageSupportOnDb(sqlite3* db)
 +---------------+---------------+---------------+---------------+---------------+------*/
 static int besqlite_db_init(sqlite3* db, char** pzErrMsg, struct sqlite3_api_routines const* pApi)
     {
+
     // and the "InVirtualSet" SQL function. It requires at least two arguments: the address of the VirtualSet and the value(s) to test
-    const auto rc = sqlite3_create_function_v2(db, "InVirtualSet", -1, SQLITE_UTF8, nullptr, &isInVirtualSet, nullptr, nullptr, nullptr);

```

Comment:
Looks unrelated to the story, but why remove const?
Also, I don't see why we only assert, to me it seems like we should check rc and fail if the function failed to register

Reply:
It's assumed they never fail unless you have an OOM error and subsequent operations will fail anyway. Though we should handle or return an error if the caller provides one from a [branch testability] (/www.tutorialspoint.com/software_testing_dictionary/branch_testing.htm) point of view even if an error has an extremely low chance of ever happening.